src/clj_http/conn_mgr.clj

```diff
@@ -152,16 +143,12 @@
                    (get-keystore-scheme-registry config)
 
                    :else regular-scheme-registry)]
-    (PoolingClientConnectionManager.
-     registry timeout java.util.concurrent.TimeUnit/SECONDS)))
-
-(def dmcpr ConnPerRouteBean/DEFAULT_MAX_CONNECTIONS_PER_ROUTE)
+    (PoolingHttpClientConnectionManager. registry)))
```

Comment:
Well, as I it understands, now you are using this constructor:
```java
PoolingHttpClientConnectionManager(Registry<ConnectionSocketFactory> socketFactoryRegistry) 
```
which according to the source code https://apache.googlesource.com/httpclient/+/64fade92dfc501918024354eb7d95ee4e17ada05/httpclient/src/main/java/org/apache/http/impl/conn/PoolingHttpClientConnectionManager.java#113
just calling another constructor
```java
PoolingHttpClientConnectionManager(Registry<ConnectionSocketFactory> socketFactoryRegistry, HttpConnectionFactory<HttpRoute,ManagedHttpClientConnection> connFactory, DnsResolver dnsResolver) 
```
with `null`s for the last two parameters, and this end ups calling the biggest constructor:
```java
PoolingHttpClientConnectionManager(Registry<ConnectionSocketFactory> socketFactoryRegistry, HttpConnectionFactory<HttpRoute,ManagedHttpClientConnection> connFactory, SchemePortResolver schemePortResolver, DnsResolver dnsResolver, long timeToLive, TimeUnit tunit) 
```
with `-1, TimeUnit.MILLISECONDS` for the last two parameters.
So I guess you can just call the biggest constructor with `null` for unknown parameter and pass `timeout` with `TimeUnit.SECONDS`.